src/app/jobs/weatherevents.job.ts

```diff
@@ -0,0 +1,55 @@
+import { Job } from '../../common/job';
+import { IContainer } from '../../common/types';
+import { TextChannel } from 'discord.js';
+import ms from 'ms';
+
+interface IWeather_Event {
+  features?: (IFeaturesEntity)[] | null;
+}
+interface IFeaturesEntity {
+  properties: IProperties;
+}
+
+interface IProperties {
+  severity: string;
+  headline: string;
+  senderName: string;
+}
+
+export class WeatherEventsJob extends Job {
+  public name: string = 'Weather Events';
+  public interval: number =  ms('1h');
+
+  private _ENDPOINT: string = 'https://api.weather.gov/alerts/active?area=FL';

```

Comment:
this should be `readonly static`

Reply:
Are you referring to all three fields or just the last one?

Reply:
I can't make name static since the Function object's prototype also contains a name and would cause a conflict, so I'll only make it readonly. I can make the others static and readonly.

Reply:
If we're going by that paradigm, though, then wouldn't it have to be implemented across the other jobs?

Reply:
I'm referring to only changing `_ENDPOINT` to a static field since it doesn't change across any instance of a plugin.

Reply:
gotcha.

Reply:
made it to where only _ENDPOINT has static readonly so it's not confusing.